tsconfig.json

```diff
@@ -19,7 +19,8 @@
       "@spartacus/core": ["projects/core/public_api"],
       "@spartacus/storefront": ["projects/storefrontlib/src/public_api"],
       "@spartacus/vendor": ["projects/vendor/public_api"],
-      "@spartacus/assets": ["projects/assets/src/public_api"]
+      "@spartacus/assets": ["projects/assets/src/public_api"],
+      "@spartacus/incubator": ["projects/incubator/src/public_api"]
```

Comment:
with core, I started to reference the public_api from the root folder, rather than the `src`. I wouldn't mind to continue that way, but it's only cosmetics.

Reply:
Yes indeed